modules/router-store/src/serializer.ts

```diff
@@ -39,9 +39,15 @@ export class DefaultRouterStateSerializer
       data: route.data,
       url: route.url,
       outlet: route.outlet,
-      routeConfig: {
-        component: route.routeConfig ? route.routeConfig.component : undefined,
-      },
+      routeConfig: route.routeConfig
```

Comment:
Shouldn't the `routeConfig` remain an empty object if its not defined to keep the existing behavior?

Reply:
Flag is as a breaking change. I'd rather the behavior be consistent and this would be the time to break it.

Reply:
Something like this? I have some troubles to properly word this.

Reply:
👍